templates/includes/base.html

```diff
@@ -4,45 +4,30 @@
         <title>Twitter</title>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-
         {% block specific_meta %}
-
         {% endblock %}
-
-
-
     </head>
     <body>
-
         <div class="container">
             <div class="row">
-
                 <div class="col-md-3">
-                    <div class="left_sidebar">
-
-                    </div>
+                    <div class="left_sidebar"></div>
```

Comment:
Do we need empty tags ?

Reply:
okay got it, my mistake. I'll remove these